source/install/use-kubernetes-mattermost.rst

```diff
@@ -82,13 +82,15 @@ Open the ``mattermost-installation.yaml`` file and add ``blueGreen`` under ``spe
 
 When the manifest is updated, two new ingresses (proxies) are created at ``blue.yourmattermosturl.com`` and ``green.yourmattermosturl.com``.
 
-To access the new ingresses, create CNAME or IP address records in your DNS registration service for the ``ingressName`` in your manifest, pointing to the address you just copied. For example, on AWS you would do this within a hosted zone in Route53. Use the required ``ingressName`` URL in your browser to directly access blue or green at any time.
+To access the new ingresses, create CNAME or IP address records in your DNS registration service for the ``ingressName`` in your
+manifest, pointing to the address you just copied. For example, on AWS you would do this within a hosted zone in Route53.
+Use the required ``ingressName`` URL in your browser to directly access blue or green at any time.
 
 To update the version of blue or green, change the version in the manifest to
 match the current version or the version you’d like to deploy. This
 change (regardless of which is the ``productionDeployment``) initiates a database migration.
-The schema is backwards and forwards compatible across minor versions  (from 5.9 onwards) and will not disrupt the production deployment.
-However, it will auto-upgrade the database. 
+The schema is backwards and forwards compatible across minor versions (from 5.9 onwards) and will not disrupt the production deployment.
+However, it will auto-upgrade the database.
```

Comment:
It seems like this could be a major change or potentially cause issues. Is there any data loss as a side-effect?

Reply:
Nope, there shouldn't any problems. The upgrade paths are tested well.